tests/lib/external/snapd-testing-tools/tests/os.query/task.yaml

```diff
@@ -145,16 +151,16 @@ execute: |
             os.query is-centos
             ! os.query is-core
             ;;
-        opensuse-15.3-64)
+        opensuse-15.4-64)
             os.query is-opensuse
-            os.query is-opensuse 15.3
+            os.query is-opensuse 15.4
+            ! os.query is-opensuse tumbleweed
             os.query is-classic
             ! os.query is-core
             ;;
-        opensuse-15.4-64)
+        opensuse-15.5-64)
             os.query is-opensuse
-            os.query is-opensuse 15.4
-            ! os.query is-opensuse tumbleweed
```

Comment:
Was removing this assertion intentional?

Reply:
yes

Reply:
for opensuse-15.5-64) now it is checking os.query is-opensuse 15.5 
and for opensuse-15.4-64) we check ! os.query is-opensuse tumbleweed